master/internal/api_command.go

```diff
@@ -100,6 +100,10 @@ func (a *apiServer) getCommandLaunchParams(ctx context.Context, req *protoComman
 	taskSpec.TaskContainerDefaults = taskContainerDefaults
 	taskSpec.AgentUserGroup = agentUserGroup
 	taskSpec.Owner = user
+	taskSpec.ExtraEnvVars = map[string]string{
```

Comment:
is there any way we can set these for all containers in `pkg/task/task.go::TaskSpec.EnvVars()`?

We definitely need these set for trial containers as well as for notebooks.